src/gfx/main.cpp

```diff
@@ -287,10 +288,25 @@ int main(int argc, char *argv[]) {
 
 	options.input = argv[argc - 1];
 
-	auto autoOutPath = [](bool autoOptEnabled, std::filesystem::path &path, char const *extension) {
+	auto autoOutPath = [](bool autoOptEnabled, std::string &path, char const *extension) {
 		if (autoOptEnabled) {
-			path = options.input;
-			path.replace_extension(extension);
+			constexpr std::string_view chars =
```

Comment:
As a replacement for `std::filesystem::replace_extension`, I'd extract all this out into a `replace_extension` utility function.